docs/tiltfile_config.md

```diff
@@ -146,12 +146,25 @@ cfg = config.parse()
 config.set_enabled_resources(cfg.get('args', []))
 ```
 
+## Flag types
+Here are some sample usages of different types of flags that are supported:
+```python
+# list of string
```

Comment:
if you keep this format, def put in "paragraph breaks" in your code block (e.g. line break before "single string".

consider
```
# single string
# invocation: tilt up -- ...
```
to make it clear what that line of comment is for (took me a second)

Reply:
##### Single String
###### Invocation: `tilt up -- ...`
```
str(whatever)
```